File: overview/rev1/guides/examples/java/credit-split.java
// bankAccountHrefA is the stored href for the bank account for Person A
BankAccount bankAccountA = new BankAccount(bankAccountHrefA);

HashMap<String, Object> payloadA = new HashMap<String, Object>();
payloadA.put("amount", 50000);
payloadA.put("description", "Payout for order #1111");

try {
    Credit credit = bankAccountA.credit(payloadA);
}
catch (HTTPError e) {}


// bankAccountHrefB is the stored href for the bank account for Person A
BankAccount bankAccountB = new BankAccount(bankAccountHrefB);

HashMap<String, Object> payloadB = new HashMap<String, Object>();
payloadB.put("amount", 50000);
payloadB.put("description", "Payout for order #1111");

try {
    Credit credit = bankAccountB.credit(payloadB);
}
catch (HTTPError e) {}

File: overview/rev1/guides/examples/csharp/credit-split.cs
// bank_account_href_a is the stored href for the BankAccount for Person A
BankAccount bank_account_person_a = BankAccount.Fetch(bank_account_href_a);
Dictionary<string, object> creditPayload = new Dictionary<string, object>();
creditPayload.Add("amount", 5000);
creditPayload.Add("description", "Payout for order #1111");
Credit credit = bank_account_person_a.Credit(creditPayload);

// bank_account_href_b is the stored href for the BankAccount for Person B
BankAccount bank_account_person_b = BankAccount.Fetch(bank_account_href_b);
Dictionary<string, object> creditPayload = new Dictionary<string, object>();
creditPayload.Add("amount", 5000);
creditPayload.Add("description", "Payout for order #1111");
Credit credit = bank_account_person_b.Credit(creditPayload);